src/core/ngx_connection.c

```diff
@@ -78,6 +78,7 @@ ngx_create_listening(ngx_conf_t *cf, struct sockaddr *sockaddr,
 
     ls->fd = (ngx_socket_t) -1;
     ls->type = SOCK_STREAM;
+    ls->protocol = 0;
```

Comment:
I guess this is not needed, because they did that before: `ngx_memzero(ls, sizeof(ngx_listening_t));`